ui/app/pages/create-account/connect-hardware/index.js

```diff
@@ -97,19 +101,19 @@ class ConnectHardwareForm extends Component {
 
           const newState = { unlocked: true, device, error: null };
           // Default to the first account
-          if (this.state.selectedAccount === null) {
+          if (this.state.selectedAccounts.length === 0) {
             accounts.forEach((a) => {
               if (a.address.toLowerCase() === this.props.address) {
                 newState.selectedAccount = a.index.toString();
               }
             });
             // If the page doesn't contain the selected account, let's deselect it
           } else if (
-            !accounts.filter(
-              (a) => a.index.toString() === this.state.selectedAccount,
+            !accounts.filter((a) =>
```

Comment:
I think we want to ensure that _all_ accounts are on the current page here. This tests that _at least one_ account is on the current page.

Reply:
Removed all this :)

Reply:
Now it preserves the selected accounts on pages that aren't visible. This could be confusing :thinking: 

But it's required to connect a set of accounts that spans multiple pages...

I guess this is fine for now, but the design is sub-optimal. We should have some indicator that there are accounts selected that aren't shown. Or maybe a list of all selected accounts? Not sure.